### DISCUSSION TITLE
Why does my dpgen automatically stop every time I iterate three times? Some structural accuracy has not reached the requirements

### DISCUSSION THREAD

**Discussion by shenghaoyun** [deepmodeling/dpgen#1590] [Q&A]
![Snipaste_2024-07-16_20-17-42](https://github.com/user-attachments/assets/0dd45654-2f8b-4d0b-b8a0-76c6d0b7a714)
Why does my dpgen automatically stop every time I iterate three times? Some structural accuracy has not reached the requirements

**Comment by njzjz** [comment]
It is decided by the number of items in `model_devi_jobs`.
